src/prefect/cli/deployment.py

```diff
@@ -411,31 +412,44 @@ async def run(
         )
     parameters = {**multi_params, **cli_params}
 
-    # Parse the start time if provided
-    if start_in:
-        start_time_raw = "in " + start_in
-    elif start_at:
-        start_time_raw = "at " + start_at
-    else:
-        start_time_raw = None
+    if start_in and start_at:
+        exit_with_error(
+            "Expected optional start_in field or start_at field but not both."
+        )
 
-    if start_time_raw:
+    elif start_in is None and start_at is None:
+        scheduled_start_time = now
+        human_dt_diff = " (now)"
+    else:
+        if start_in:
+            start_time_raw = "in " + start_in
+        else:
+            start_time_raw = "at " + start_at
         with warnings.catch_warnings():
             # PyTZ throws a warning based on dateparser usage of the library
             # See https://github.com/scrapinghub/dateparser/issues/1089
             warnings.filterwarnings("ignore", module="dateparser")
-            start_time_parsed = dateparser.parse(
-                start_time_raw,
-                settings={
-                    "TO_TIMEZONE": "UTC",
-                    "RELATIVE_BASE": datetime.fromtimestamp(now.timestamp()),
-                },
-            )
+
+            try:
+                start_time_parsed = dateparser.parse(
+                    start_time_raw,
+                    settings={
+                        "TO_TIMEZONE": "UTC",
+                        "RELATIVE_BASE": datetime.fromtimestamp(now.timestamp()),
+                    },
+                )
+            except Exception as exc:
+                exit_with_error(
+                    f"Exception occurred while parsing '{start_time_raw}': {exc!r}"
+                )
```

Comment:
I'd lean towards "Failed to parse ..." for user-facing messages. When we say "Exception occurred" we'll usually want to include a traceback.

I'd also probably use the string for the exception rather than the repr. What kind of exceptions occur here / with what sort of messages?

Reply:
Sounds good. These are the exceptions that could occur according to their documentation:

> ValueError: Unknown Language, TypeError: Languages argument must be a list, SettingValidationError: A provided setting is not valid.

Based on that, should we expect that `start_time_parsed` will always be set to `None` if it's parsed incorrectly and that these errors are too unlikely to be concerned about?

Reply:
We're in control over the language / settings being passed in, so I think that these errors don't need to be captured — there's nothing the user can do if these occur.